packages/venia-ui/lib/components/Header/searchTrigger.css

```diff
@@ -1,13 +1,36 @@
+:root {
+    --search-triger-width: 3rem;
```

Comment:
I think it's pretty dangerous to add global custom properties for a component. Namespacing will quickly become an issue.
```suggestion
```

Reply:
Removed